packages/server/src/fhir/search.ts

```diff
@@ -1292,6 +1288,39 @@ function buildDateSearchFilter(table: string, impl: ColumnSearchParameterImpleme
   return new Condition(new Column(table, impl.columnName), fhirOperatorToSqlOperator(filter.operator), filter.value);
 }
 
+/**
+ * Builds a quantity search filter.
+ * @param table - The resource table name.
+ * @param impl - The search parameter implementation info.
+ * @param filter - The search filter.
+ * @returns The select query condition.
+ */
+function buildQuantitySearchFilter(
+  table: string,
+  impl: ColumnSearchParameterImplementation,
+  filter: Filter
+): Expression {
+  const [number, _system, _code] = splitN(filter.value, '|', 3);
+  if (!number) {
+    throw new OperationOutcomeError(badRequest('Invalid quantity value: ' + filter.value));
+  }
+
+  if (filter.operator === Operator.APPROXIMATELY) {
+    // Search for operators within 10% of the value
+    // See: https://hl7.org/fhir/R4/search.html#prefix
+    // 	 The value for the parameter in the resource is approximately the same to the provided value.
+    //   Note that the recommended value for the approximation is 10% of the stated value
```

Comment:
Interesting, 10% seems super arbitrary... 

Intuitively I wouldn't say 81 is approximately 90, but eh, I guess stick with what the spec recommends